Packs/PAN-OS/Integrations/Panorama/Panorama.py

```diff
@@ -7613,6 +9640,31 @@ def main():
         elif demisto.command() == 'panorama-install-file-content-update':
             panorama_install_file_content_update_command(args)
 
+        elif demisto.command() == 'pan-os-platform-get-arp-tables':
+            topology = get_topology()
+            result = get_arp_tables(topology, **demisto.args())
+            dataclasses_to_command_results(result)
```

Comment:
call dataclasses_to_command_results function inside the command function (in this case get_arp_tables)

Reply:
This breaks the design of those functions. We want them to return the dataclass instances, not be responsible for the interface to XSOAR. This should make this code much more maintainable in the future.